### PR TITLE
Add unstableLabelStyle

### PR DESCRIPTION
**Pull request by Slapbox**
# Test Plan

Tested the `Input.tsx` `unstableLabelStyle` but did not separately test the same changes in `Switch.tsx`.

@mmazzarolo thoughts on naming convention and on the descriptions in the readme? Any other notes?

### CODE REVIEW

File: src/Modal.tsx [hunk 1]
```diff
@@ -145,12 +145,8 @@ export class Modal extends Component<ModalProps, ModalState> {
   };
 
   render() {
-    const {
-      children,
-      onBackdropPress,
-      contentStyle,
-      ...otherProps
-    } = this.props;
+    const { children, onBackdropPress, contentStyle, ...otherProps } =
```
Comment (Slapbox) [inline src/Modal.tsx:148]: Also includes this unrelated change to satisfy ESLint complaint.

**Review by mmazzarolo — COMMENTED** [review]
PR looks solid to me! Thanks! Let's fix and merge #141 first, and then we'll ship this one as well 👍

**Comment by Slapbox** [comment]
@mmazzarolo sorry for the huge delay - I'm finally getting around to this! See #150.
